staging/src/k8s.io/apiserver/pkg/util/apihelpers/helpers.go

```diff
@@ -98,3 +104,13 @@ func (s FlowSchemaSequence) Less(i, j int) bool {
 func (s FlowSchemaSequence) Swap(i, j int) {
 	s[i], s[j] = s[j], s[i]
 }
+
+var metaInternalVersionCodecsWithCBOR = serializer.NewCodecFactory(metainternalversionscheme.Scheme, serializer.WithSerializer(cbor.NewSerializerInfo))
+
+// todo
```

Comment:
fill out this TODO.  I think this is so that we can encode DeleteOptions as CBOR?

Reply:
But something made you move it here?  why here instead of the `metainternalversionscheme` package?

Reply:
It's here because I can't import the package that contains the apiserver feature gate instance from apimachinery.